certified-connectors/Cradl AI/Readme.md

```diff
@@ -0,0 +1,42 @@
+## Cradl AI
+
```

Comment:
Hello @staalezh, 

Could you please update the readme.md file as per the template in the below pat?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Hi @vmanoharas,

Absolutely, I just updated the Readme. :-)